utils/install_docker.py

```diff
@@ -0,0 +1,26 @@
+import os
+import sys
+
+sys.path.append(os.path.normpath(os.path.join(os.path.dirname(os.path.abspath(__file__)), "..")))
+
+from fame.core import fame_init
+from utils.initial_data import create_initial_data
+from utils.install import create_admin_user, create_user_for_worker, add_community_repository
+
+
+def main():
+    fame_init()
+
+    print("[+] Creating initial data ...")
+    create_initial_data()
+
+    print("[+] Creating users...")
+    create_admin_user()
+    create_user_for_worker(dict())
+
+    if os.getenv("FAME_INSTALL_COMMUNITY_REPO", "1") == "1":
+        add_community_repository()
+
+
+if __name__ == "__main__":
+    main()
```

Comment:
Could you explain why we would not be able to use the main install script ?

Reply:
When we are in the docker container, we only need to prepare the database. The configuration will be generated from environment variables and all requirements are already installed because they are static. Also, we don't need to drop the configuration to disk.
This is just to remove duplicate/unnecessary steps when booting up the container.